packages/m2m-gateway/.env

```
@@ -0,0 +1,34 @@
+HOST=0.0.0.0
+PORT=4000
+LOG_LEVEL=info
+M2M_GATEWAY_INTERFACE_VERSION="2.0"
+WELL_KNOWN_URLS="http://127.0.0.1:4500/jwks.json"  
+ACCEPTED_AUDIENCES="dev.interop.pagopa.it/m2m,dev.interop.pagopa.it/m2m"  
```

Duplicated, maybe we have this in all services 🤔  Let's fix it here

```suggestion
ACCEPTED_AUDIENCES="dev.interop.pagopa.it/m2m"  
```